File: simple-dpgf-api/src/main/java/com/simpledpgfapi/user/model/organization/dto/OrganizationDto.java
package com.simpledpgfapi.user.model.organization.dto;

import com.simpledpgfapi.user.model.organization.OrganizationTypeEnum;
import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class OrganizationDto {
    @NotNull
    private String id;
    @NotNull
    private String name;
    @NotNull
    private OrganizationTypeEnum organizationType;
    private Double memberLicenseCounter;
    private Double maxMemberLicenseCounter;
    private Double projectLicenseCounter;
    private Double maxProjectLicenseCounter;
}

File: simple-dpgf-webfront/src/app/core/dtos/organization/OrganizationDto.ts
import { OrganizationTypeEnum } from "../../enums/OrganizationTypeEnum";

export interface OrganizationDto {
  id: string;
  name: string;
  organizationType: OrganizationTypeEnum;
  memberLicenseCounter: number;
  maxMemberLicenseCounter: number;
  projectLicenseCounter: number;
  maxProjectLicenseCounter: number;
}
